### ISSUE TITLE
clean up `isOld` 

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/portal-ui#1497]
Once the transition to the new prov api is complete, we should get rid of the transition code.

see: https://github.com/hubmapconsortium/portal-ui/pull/1487#discussion_r567893166_

**Comment by john-conroy** [comment]
Seems to be a duplicate of #1541. Re-open if not.
